### PR TITLE
Fix vertical centering of navbar logo

### PR DESCRIPTION
**Pull request by jb3**
Commit 6d53df0c8bf24b44be76a8f0ef82ade86079f6f6 tried to address the logo leaking out of the navbar but incidentally seems to have pinned it to the top of the navbar.

Removing all manual positioning seems to leave Bootstrap happy enough to style it and so with just `max-height: 40px;` I see the following behavior in Firefox and Chromium.

image

```diff
diff --git a/templates/base.html b/templates/base.html
index 9c73904..3cd53e1 100644
--- a/templates/base.html
+++ b/templates/base.html
@@ -36,7 +36,7 @@
 <nav class="navbar navbar-expand-lg navbar-dark bg-dark" role="navigation">
   <div class="container">
       <a class="navbar-brand" href="{% if request.user.is_authenticated %}https://rigs.nottinghamtec.co.uk{%else%}https://nottinghamtec.co.uk{%endif%}">
-        <img src="{% static 'imgs/logo.webp' %}" class="mr-auto" style="max-height: 40px; position: absolute; left: 0.5em; top: 0;" alt="TEC's Logo: Serif 'TEC' vertically next to a blue box with the words 'PA and Lighting', surrounded by graduated rings" id="logo">
+        <img src="{% static 'imgs/logo.webp' %}" class="mr-auto" style="max-height: 40px;" alt="TEC's Logo: Serif 'TEC' vertically next to a blue box with the words 'PA and Lighting', surrounded by graduated rings" id="logo">
       </a>
       {% block titleheader %}
       {% endblock %}
```

I can't really figure out what would cause the rendering discrepancies on other platforms.

### CODE REVIEW

**Comment by FreneticScribbler** [comment]
I'm not particularly a fan of it not being right in the far left corner though, hence the absolute position hack. Definitely agree it needs another look though :thinking:

**Comment by jb3** [comment]
On my version it is in-line with other nav items, not in the far left corner?

image

**Comment by FreneticScribbler** [comment]
Sorry. Double negative made that confusing let me try again. My original intent was to have it in the far left corner, but because the navbar is only 80vw I had to be a bit hacky to get it there. 

Open to the idea it'd be better where you have put it (certainly easier), but personally reluctant to adjust the original navbar layout.

**Comment by jb3** [comment]
I have no preference really, I'll have a play around and see if I can get the old style with left pinning but with vertical centering, and without the overlapping that Isaac found yesterday on mobile devices:

image

**Comment by jb3** [comment]
I can get it working nicely on desktop but it doesn't play nice with the extending nav on mobile at all, the woes of responsive design.

**Comment by FreneticScribbler** [comment]
Yup, trying to design against so many targets is *pain*. I'll take another look myself at some point ...

**Comment by jb3** [comment]
Closing for now and going to merge this fix in from another branch.

If you want to move it back to far left at some point in future that's fine but just going to merge this so that the logo is not actively overlapping with content on mobile devices.
